pkg/skaffold/lint/dockerfiles.go

```diff
@@ -129,6 +128,11 @@ func GetDockerfilesLintResults(ctx context.Context, opts Options, dockerCfg dock
 		for _, a := range c.Build.Artifacts {
 			if a.DockerArtifact != nil {
 				ws := filepath.Join(workdir, a.Workspace)
+				// TODO(aaron-prindle) HACK - multi-module configs use abs path for a.Workspace vs single module which has rel path
+				// see if there is a built-in/better way of handling this.  This is currently working for multi-module
+				if filepath.IsAbs(a.Workspace) {
+					ws = filepath.Join(a.Workspace)
```

Comment:
should this be just `ws = a.Workspace`?
```suggestion
					ws = a.Workspace
```

Reply:
done